src/redux/action-types/resetPassword.js

```diff
@@ -0,0 +1,8 @@
+export const REQUEST_RESET = 'REQUEST_RESET';
+export const REQUEST_RESET_SENT = 'REQUEST_RESET_SENT';
+export const REQUEST_RESET_FAIL = 'REQUEST_RESET_FAIL';
+export const OPEN_RESET = 'OPEN_RESET';
```

Comment:
@niyobobo I think this action type is not used.

Reply:
Solved it